### PR TITLE
add doc for dockerfilePath

### PR DESCRIPTION
**Pull request by vbehar**
added in https://github.com/openshift/origin/pull/5976 (not yet merged in origin)

### CODE REVIEW

File: rest_api/openshift_v1.adoc [hunk 1]
```diff
@@ -10284,6 +10284,7 @@ ObjectReference contains enough information to let you inspect or modify the ref
 |noCache|if true, indicates that the Docker build must be executed with the --no-cache=true flag|false|boolean|false
 |env|additional environment variables you want to pass into a builder container|false|<<v1.EnvVar>> array|
 |forcePull|forces the source build to pull the image if true|false|boolean|false
+|dockerfilePath|path of the Dockerfile to use for building the Docker image, relative to the contextDir, if set|false|string|
```
Comment (php-coder) [inline rest_api/openshift_v1.adoc]: Is it path or file name?

Comment (vbehar) [inline rest_api/openshift_v1.adoc]: Path. Could be somedir/somedockerfile.
Le 15 déc. 2015 15:46, "Vyacheslav Semushin" <notifications@github.com> a
écrit :

**Comment by bparees** [comment]
lgtm, @openshift/team-documentation ptal.

File: dev_guide/builds.adoc [hunk 2]
```diff
@@ -327,6 +327,32 @@ The following options are specific to the
...
+{
+  "strategy": {
+    "type": "Docker",
+    "dockerStrategy": {
+      "dockerfilePath": "dockerfiles/app1"
+    }
+  }
+}
+----
+====
```
Comment (bfallonf) [inline dev_guide/builds.adoc]: Any chance this example could be in yaml and not json? We're trying to lean towards that in the docs.

**Comment by bfallonf** [comment]
@vbehar @bparees One comment, but other than that, all good :cool:

**Comment by bparees** [comment]
@bfallonf feature has merged.  is conversion to yaml something your team can do?  http://jsontoyaml.com/

File: dev_guide/builds.adoc (hunk 2 above)
Comment (vbehar) [inline dev_guide/builds.adoc]: I initially wrote it in JSON because the others examples in the page are in JSON. But I just changed it to YAML.

**Comment by bfallonf** [comment]
@bparees @vbehar  Going back and changing all our examples from json to yaml is something on the backlog. We're trying to encourage any new additions to be in yaml. @vbehar thanks for making the change!

Question, though: Is this new content? Should this be just for 3.1 or even 3.2? I'm ok to merge this, but depending on which branch it should head into.

**Comment by bparees** [comment]
It'll definitely be in 3.2, it's not in 3.1 yet, but depending what gets
picked up, could be soon.

Ben Parees | OpenShift

**Comment by bfallonf** [comment]
Ok. Might be best to wait until this merges then. I'll label it accordingly. Thanks, all.

**Comment by adellape** [comment]
@bfallonf Switched milestone to OSE Future til we can verify if it's landing in 3.1.1.

**Comment by adellape** [comment]
@vbehar Can you squash this into one commit?

**Comment by vbehar** [comment]
ok I rebased and squashed everything into 1 commit

**Comment by adellape** [comment]
:+1: Thanks.

**Pull request merged**